templates/aws-vpc.template

```diff
@@ -193,154 +193,154 @@
+            "Description": "CIDR block for the public DMZ subnet 2 located in Availability Zone 2.",
             "Type": "String"
         },
         "PublicSubnet3CIDR": {
             "AllowedPattern": "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\\/(1[6-9]|2[0-8]))$",
             "ConstraintDescription": "CIDR block parameter must be in the form x.x.x.x/16-28",
             "Default": "10.0.160.0/20",
-            "Description": "CIDR block for the public DMZ subnet 3 located in Availability Zone 3",
+            "Description": "CIDR block for the public DMZ subnet 3 located in Availability Zone 3.",
             "Type": "String"
         },
         "PublicSubnet4CIDR": {
             "AllowedPattern": "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\\/(1[6-9]|2[0-8]))$",
             "ConstraintDescription": "CIDR block parameter must be in the form x.x.x.x/16-28",
             "Default": "10.0.176.0/20",
-            "Description": "CIDR block for the public DMZ subnet 4 located in Availability Zone 4",
+            "Description": "CIDR block for the public DMZ subnet 4 located in Availability Zone 4.",
             "Type": "String"
         },
         "PublicSubnetTag1": {
             "AllowedPattern": "^([a-zA-Z0-9+\\-._:/@]+=[a-zA-Z0-9+\\-.,_:/@ *\\\\\"'\\[\\]\\{\\}]*)?$",
             "ConstraintDescription": "tags must be in format \"Key=Value\" keys can only contain [a-zA-Z0-9+\\-._:/@], values can contain [a-zA-Z0-9+\\-._:/@ *\\\\\"'\\[\\]\\{\\}]",
             "Default": "Network=Public",
-            "Description": "tag to add to public subnets, in format Key=Value (Optional)",
+            "Description": "(Optional) Tag to add to public subnets, in format Key=Value.",
             "Type": "String"
         },
         "PublicSubnetTag2": {
             "AllowedPattern": "^([a-zA-Z0-9+\\-._:/@]+=[a-zA-Z0-9+\\-.,_:/@ *\\\\\"'\\[\\]\\{\\}]*)?$",
             "ConstraintDescription": "tags must be in format \"Key=Value\" keys can only contain [a-zA-Z0-9+\\-._:/@], values can contain [a-zA-Z0-9+\\-._:/@ *\\\\\"'\\[\\]\\{\\}]",
             "Default": "",
-            "Description": "tag to add to public subnets, in format Key=Value (Optional)",
+            "Description": "(Optional) Tag to add to public subnets, in format Key=Value.",
             "Type": "String"
         },
         "PublicSubnetTag3": {
             "AllowedPattern": "^([a-zA-Z0-9+\\-._:/@]+=[a-zA-Z0-9+\\-.,_:/@ *\\\\\"'\\[\\]\\{\\}]*)?$",
             "ConstraintDescription": "tags must be in format \"Key=Value\" keys can only contain [a-zA-Z0-9+\\-._:/@], values can contain [a-zA-Z0-9+\\-._:/@ *\\\\\"'\\[\\]\\{\\}]",
             "Default": "",
-            "Description": "tag to add to public subnets, in format Key=Value (Optional)",
+            "Description": "(Optional) Tag to add to public subnets, in format Key=Value.",
```

Comment:
Rename Optional Tag 3

Reply:
Added tag numbers to the labels for all subnet tags (private A, private B, public).